### ISSUE TITLE
Live Preview from command line

### ISSUE THREAD

**Issue by daveshook** [microsoft/vscode-livepreview#172]
tldr; I would like to preview an html file from the command line but when I run command `gp preview $(gp url 3000)/index.html`, I get an error.

Details:
If I run command `gp preview $(gp url 3000)/index.html` after first starting the gitpod workspace, when the browser tab opens with `https://3000-plum-galliform-dpkqmaaq.ws-us18.gitpod.io/cluster-linking-exercise.html` in the address bar and the window contains the message `Port 3000 Not Found Please make sure this port is exposed and your application is up and running.`

If I then open the file in the editor and preview it using the `Live Preview: Show Preview` button, the file is displayed in the new preview window after a short delay until port 3000 is activated.

If I then return to the window that had the error, the file preview is now visible.

If I close the editor and preview windows and then run command `gp preview $(gp url 3000)/index.html` again, the file preview is shown. If I close the preview tab and wait, after a bit of delay port 3000 returns to an inactive state. At that point, running the command gives the same error that was previously mentioned.

Enhancement request:
Could this extension be updated so that it can be activated from the command line in the same manner as it is when the `Live Preview: Show Preview` button is pushed?

**Comment by andreamah** [comment]
In terms of the GitPod integration, unfortunately, that's out of scope since this might be an issue with how GitPod works with VS Code or the extension. If you see the same issues on somewhere like codespaces (which is supported), please let me know.

The enhancement request is a duplicate of #368
